<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LH7U05CG7493V54" name="test.java.TestCSRSearchRemittance" tests="1" failures="0" timestamp="24 Oct 2018 11:56:52 GMT" time="18.290" errors="1">
  <testcase name="testTypeAndPaymentStatusForVCP" time="18.290" classname="test.java.TestCSRSearchRemittance">
    <error type="java.lang.NoClassDefFoundError" message="org/apache/poi/xssf/usermodel/XSSFWorkbook">
      <![CDATA[java.lang.NoClassDefFoundError: org/apache/poi/xssf/usermodel/XSSFWorkbook
	at main.java.Utils.TestDataReader.readFile(TestDataReader.java:286)
	at main.java.Utils.TestDataReader.<init>(TestDataReader.java:45)
	at main.java.nativeFunctions.TestBase.cacheTestDataReaderObject(TestBase.java:228)
	at main.java.Utils.DataBase.executeSelectQuery(DataBase.java:118)
	at main.java.pageObjects.SearchRemittanceSearchCriteria.dataProvider(SearchRemittanceSearchCriteria.java:390)
	at main.java.pageObjects.SearchRemittanceSearchCriteria.getPaymentStatusFromDB(SearchRemittanceSearchCriteria.java:341)
	at test.java.TestCSRSearchRemittance.testTypeAndPaymentStatusForVCP(TestCSRSearchRemittance.java:318)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: org.apache.poi.xssf.usermodel.XSSFWorkbook
	at java.net.URLClassLoader.findClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	... 32 more
]]>
    </error>
  </testcase> <!-- testTypeAndPaymentStatusForVCP -->
</testsuite> <!-- test.java.TestCSRSearchRemittance -->
